script.js

```diff
@@ -1,11 +1,24 @@
 // Create an array called `colors` that contains three colors.
+var colors = ["blue","red","yellow"]
 
 // Log the last item in the array to the console.
+console.log (colors[0])
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (var i = 0; i < colors.length; i++) {
+  console.log ("My favorite color is" + " " + (colors[0]));
+  console.log ("My favorite color is" + " " + (colors[1]));
+  console.log ("My favorite color is" + " " + (colors[2]));
+}
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor =
+{
+  gender:"Male",
+  mindset:"500",
+  swag:"On-point"
+};
 // Add a `facial-hair` property to `instructor`.
+instructor.facial_hair = "Beard";
```

Comment:
The key needs to have a hyphen, like so `facial-hair`. In order to set that property, you need to do something like this...

```js
instructor["facial-hair"] = true
```